frontends/election-manager/src/screens/ballot_screen.tsx

```diff
@@ -271,9 +302,11 @@ export function BallotScreen(): JSX.Element {
               Back to List Ballots
             </LinkButton>
           </p>
-          <p>
-            Ballot Package Filename: <Monospace>{filename}</Monospace>
-          </p>
+          {!draftMode && (
+            <p>
+              Ballot Package Filename: <Monospace>{filename}</Monospace>
+            </p>
+          )}
```

Comment:
Do you know why we show this filename on this screen? The action to export the ballot package is on the parent screen.

Reply:
I just looked in the git blame and it looks like the filename display was shown before we added the feature to save individual ballots as PDFs. So, at the time, the Ballot Package was the only place to access the PDF. Now that we have a few other, easier ways, I think it should be removed.

Reply:
Thanks for finding that explanation @adghayes! Given that context, I also lean toward removing. @beausmith Any concerns with removing? (I won't block this PR on that call)